imaginairy/cli/upscale.py

```diff
@@ -40,27 +31,15 @@ def validate_ext(ctx, param, value):
     "--upscale-model",
     multiple=True,
     type=str,
-    default=["realesrgan"],
+    default=["ultrasharp"],
     show_default=True,
     help="Specify one or more upscale models to use.",
 )
 @click.option(
     "--format",
-    default="{original}_upscaled_{number}_{algorithm}_{architecture}_{date}_{time}.{ext}",
-    type=str,
-    help="Format file name.",
-)
-@click.option(
-    "--date",
-    callback=validate_date,
-    help="Specify the date format, used with --format.",
-    type=str,
-)
-@click.option(
-    "--ext",
-    callback=validate_ext,
-    help="Specify the file extension, used with --format.",
+    default="{original}_upscaled_{number}_{algorithm}_{date:%Y-%m-%d}_{time:%H-%M-%S}",
     type=str,
+    help="Format file name using {original},{number},{algorithm},{date}, and {time}.",
```

Comment:
add explanation of what syntax is and how to do zero padding and date formatting

Reply:
Added zero padding and syntax explanations.